src/main/java/MainApp.java

```diff
@@ -19,7 +20,7 @@ public static HttpServer startServer() {
     final ResourceConfig config = new ResourceConfig();
 
     config.register(MyResource.class);
-
+    config.register(NasaResource.class);
    // config.register(AutoScanFeature.class);
```

Comment:
Let's remove commented out code. Usually it is better to not have it so our code is cleaner. In case we need something from before, we can always use git to recover.